README.md

```diff
@@ -32,6 +32,17 @@ docker run -it -v $srcDir:/src  <DOCKER_NAME>:<DOCKER_VERSION>
 We use the [codacy-plugins-test](https://github.com/codacy/codacy-plugins-test) to test our external tools integration.
 You can follow the instructions there to make sure your tool is working as expected.
 
+## Generating documentation
+
+Update brakeman version in `docs/Gemfile` and then run:
```

Comment:
Can you check the way we do it on rubocop? There we use the same file to setup the tool locally and inside the docker.